### ISSUE TITLE
Rename binary to avoid confusion

### ISSUE THREAD

**Issue by Widdershin** [Widdershin/sonic-pi-cli#4]
Sonic Pi ships a binary on Linux named `sonic-pi`. To avoid confusion, I will rename my binary to something with a clear difference (currently `sonic_pi`). I am not a huge fan of `sonic_pi_cli`, as I feel the `cli` is a bit redundant in a command line context. If anyone has any suggestions for the new binary name, I'd like to hear them.

**Comment by lpil** [comment]
:+1:

**Comment by lpil** [comment]
`sonic-pi-send` ?

**Comment by dermusikman** [comment]
`sonic-ping`

**Comment by davidair** [comment]
sonic-play?

**Comment by rbnpi** [comment]
I am very happy with it as it is. I have used it extensively using the present name without problem, and  would be loathe to have to update many files if it did change.

**Comment by lpil** [comment]
Just create a symlink from the old name the new and then you wouldn't need to update them.

**Comment by rbnpi** [comment]
Yes I could do that, but I haven’t found the existing name a problem. It’s easy to remember sonic-pi for the app and sonic_pi for the cli.

**Comment by lpil** [comment]
That seems very confusing to me.

**Comment by lpil** [comment]
That seems very confusing to me.

I use `sonic-pi-tool`, which will start the server for you, avoiding this issue :)

**Comment by rbnpi** [comment]
Well I’ve lived with it happily for the 2.5 years I’ve been using this cli. But each to his own.

**Comment by rbnpi** [comment]
If it is deemed necessary to have a change I would plump for sonic-pi-cli, which says what it is and is related to the app it applies to, and relates to the gem name. It would mean more typing though if using from the command line in a terminal.
However I use it embedded more often than not in other programs, eg my jukebox program for Sonic Pi, although  perhaps less so now that SP supports incoming OSC messages.
